src/finding/summary.go

```diff
@@ -137,3 +179,73 @@ func findingAttributeValue(finding types.ImageScanFinding, name string) string {
 	}
 	return ""
 }
+
+const legacyCVEURL = "https://cve.mitre.org/cgi-bin/cvename.cgi?name="
+const updatedCVEURL = "https://www.cve.org/CVERecord?id="
+
+func fixFindingURI(name string, uri string) string {
+	correctedURI := uri
+
+	// transition from the old CVE site that is deprecated
+	if strings.HasPrefix(correctedURI, legacyCVEURL) {
+		correctedURI = strings.Replace(correctedURI, legacyCVEURL, updatedCVEURL, 1)
+	}
```

Comment:
I generally don't do this, as if you later insert another predicate above you can introduce a bug. It's defensive in that way. I agree that it doesn't change behaviour, and I certainly considered it.

I don't think it would be wrong to do as suggested, I've more fallen into writing it as I did above across time and experience of making silly mistakes.